src/components/dismissible-box/dismissible-box.d.ts

```diff
@@ -12,6 +12,11 @@ export interface DismissibleBoxProps extends SpaceProps {
   onClose: React.MouseEventHandler<HTMLButtonElement>;
   /** Set the base color variant */
   variant?: "light" | "dark";
+  /** Use this prop to override the default width. Numbers from 0-1 are converted to percentage widths. Numbers greater
```

Comment:
As this component just spreads the `rest` onto the `StyledDismissibleBox` which is just a `Box` with some additional styling, I wonder if it would be better to just `extend BoxProps` and `Omit` the props we don't surface here instead (eg `display` and `justifyContent`)?

Reply:
I have no issues at all doing it that way, as long as it won't interfere with the up and coming TS refactor work.

Reply:
Sound good to me ^^^ It won't interfere with the ts refactor 👍🏻

Reply:
Well seen as @samtjo has put his 🦭 of approval on it, I'll get the changes made. Cheers lads 😄